### ISSUE TITLE
conf.json으로 현재 실행 상태 지정.

### ISSUE THREAD

**Issue by ArtemisDicoTiar** [eubinecto/wisdomify#43]
## Why?
데이터 및 모델의 버전 관리를 위한 코드가 추가되었다. 하지만 여전히 코드 상에서 수정해야하는 상태이다.
따라서 버전을 건드릴 때마다 코드를 조금씩은 계속 건드려야한다.
하지만 이전 나의 프로젝트에서 보면 이런식으로 코드내에서 버전이나 파라미터들을 건드리면 항상 어딘가 꼬여서 에러를 한번쯤은 뱉었다.

## What?
conf.json에서 실행 환경을 지정해서 
  어떤 시스템에서나 모델 버전, 데이터 버전을 자유롭게 올리고 내리고를 할 수 있고 
  모든 시스템에서 동일한 결과가 나올수 있도록 하려고 한다.

## TODOs
- [x] conf.json 스키마 설정
- [x] conf.json 로드하는 파이썬 파일 만들기
- [x] 기존에 있는 파일들 버전에 맞게 로드하도록 수정.
  - [x] paths.py
  - [x] datasets.py (?)
  - [x] main
    - [x] eval.py
    - [x] infer.py
    - [x] train.py
    - [x] deploy.py (? → 여기는 버전별로 다 로드 시켜야할 거 같은데...?)

**Comment by ArtemisDicoTiar** [comment]
## conf.json 스키마
### 주의사항
"큰 따옴표"가 있으면 해당 값은 String.
없으면 Integer, Float 등의 Number입니다.
~~~json
{
  "run_env": "{실행할 프로젝트 버전}",
  "versions": {
    "0": {
      "bert_model": "beomi/kcbert-base",
      "desc": "The first minimal-viable-product of wisdomify",
      "data_version": "0",
      "k": 11,
      "lr": 0.00001,
      "max_epochs": 20,
      "batch_size": 40,
      "repeat": 20,
      "num_workers": 4,
      "shuffle": true,
      "train_ratio": 0.8,
      "test_ratio": 0.2
    },
    "모델 버전": {
      "bert_model": "모델 트레이닝에 사용된 베이스 BERT model",
      "desc": "해당 모델 버전 간략한 설명",
      "data_version": "해당 모델 트레이닝에 사용된 데이터 버전",
      "k": 최대 길이 설정,
      "lr": learning rate 설정,
      "max_epochs": 최대 에폭값,
      "batch_size": 배치사이즈,
      "repeat": 반복 횟수,
      "num_workers": 워커 개수,
      "shuffle": 셔플여부 (이건 **Boolean** 입니다.),
      "train_ratio": 트레이닝에 사용될 데이터 비율 (0~1),
      "val_ratio": validation에 사용될 데이터 비율 (0~1),
      "test_ratio": 테스트에 사용될 데이터 비율 (0~1)
    }
  }
}
~~~

**Comment by ArtemisDicoTiar** [comment]
일단 간단하게 conf를 로드하는 파이썬 파일을 생성했다.

**Comment by ArtemisDicoTiar** [comment]
### `paths.py`
수정중인데, 체크포인트 파일명이 조금 걱정이다.
지금처럼 로스, 에폭 이런걸 다 담고 저장하면 conf.json을 수정해야한다.
일단은 모든 파일이 wisdomify_def.ckpt 혹은 wisdomify_eg.ckpt로 저장된다고 하자.
(사실 에폭수랑 로스는 로거가 텐서보드파일로 뽑아 줄거라 파일명에 없어도 될거 같은 생각이 든다)

**Comment by ArtemisDicoTiar** [comment]
main로직에 version을 argument로 받는다.
conf.json에 없어도 될듯...?

**Comment by ArtemisDicoTiar** [comment]
config.py도 없어도 되겠다.

**Comment by ArtemisDicoTiar** [comment]
트레이닝 확인중

**Comment by ArtemisDicoTiar** [comment]
트레이닝 확인완료.
eval 코드 뭔가 안된다
